File: EJERCICIOS KATTIS/IsItHalloween.com/IsItHalloween_com.java
import java.io.*;

public class IsItHalloween_com
{
    public static void main(String[] args) throws IOException
    {
        BufferedReader lc = new BufferedReader (new InputStreamReader(System.in));
        String date = lc.readLine().toUpperCase();
        String date_split[] = date.split(" ");
        if(date_split[0].equals("OCT") || date_split[0].equals("DEC"))
        {
            if(date_split[1].equals("31") || date_split[1].equals("25"))
            {
                System.out.println("yup");
            }
            else
            {
                System.out.println("nope");
            }
        }
        else
        {
            System.out.println("nope");
        }
    }
}

File: EJERCICIOS KATTIS/IsItHalloween.com/IsItHalloween.com.py

date = input().upper()
date_split = date.split()
if date_split[0] == "OCT" or date_split[0] == "DEC":
    if date_split[1] == "31" or date_split[1] == "25":
        print("yup")
    else:
        print("nope")
else:
    print("nope")

